### ISSUE TITLE
Configuration normalization

### ISSUE THREAD

**Issue by SlimeDog** [PaulAnthonyReitz/ReitzMMO#2]
Please consider normalizing the content of MonsterConfig.yml

Current version (BLAZE as an example)
```
# This config is used to set all monster related configurations
BLOCKS-PER-MOB-LEVEL: 100
ATTACK-GAIN-PER-LEVEL: 1
HEALTH-GAIN-PER-LEVEL: 1
NAMEPLATES_ENABLED: true
APPLY-ON-SPAWNER-SPAWNS: true
BLAZE: true
BLAZE_BASE_HP: 3
BLAZE_BASE_ATTACK: 3
BLAZE_SPEED: 1
BLAZE_MIN_LEVEL: 1
BLAZE_NAMEPLATES_ENABLED: true
```

could be a lot more readable as
```
# This config is used to set all monster related configurations
general:
  blocks-per-mob-level: 100
  attack-gain-per-level: 1
  health-gain-per-level: 1
  apply-on-spawner-spawns: true
  nameplates_enable: true
BLAZE:
  enabled: true
  base_hp: 3
  base_attack: 3
  speed: 1
  min_level: 1
  nameplates_enable: true
```

**Comment by RitzDaCat** [comment]
c30c27063d5bb4eceb76f39ecc33a87cf8194f9c
f53836312bb091057e3b716518c77ad55ad04d79

**Comment by SlimeDog** [comment]
Confirmed fixed in v 1.15. Thanks.
